network/wsNetwork.go

```diff
@@ -173,12 +173,9 @@ const (
 type GossipNode interface {
```

Comment:
minor nit but should we move this interface out to a non-implementation specific file?